modules/ROOT/pages/dashboards/index.adoc

```diff
@@ -0,0 +1,36 @@
+[[dashboards]]
+= About Neo4j Aura dashboards
+:description: Dashboards as a part of the new Aura console experience.
+
+Neo4j Aura dashboards help you visualize your data in a low-code manner.
+You can compose different visualizations such as tables or bar charts in dashboard pages to have relevant data at a glance.
+
+== Features
+
+* Neo4j Aura dashboards are a part of the Aura console experience
+* Neo4j Aura dashboards operate directly on your graph data via Cypher queries
+* Manage and customize your dashboards, dashboard pages and cards
+* Customize visualizations via settings and their corresponding Cypher query
+// * Parameterize visualizations or entire dashboards
+// * Apply rule-based styling to your visualizations
+// * Share your dashboards across your team or your company
+
+
+== Aura dashboards vs NeoDash commercial
```

Comment:
I don't think we should position NeoDash commercial here. 
For our Aura users the new Dashboards app will be the suggested experience, so introducing NeoDash could be confusing.